keps/sig-node/0014-runtime-class.md

```diff
@@ -176,6 +176,69 @@ const (
 An unspecified RuntimeClassName `""` is equivalent to the `legacy` RuntimeClass, though the field is
 not defaulted to `legacy` (to leave room for configurable defaults in a future update).
 
+#### Examples
+
+Suppose we operate a cluster that lets users choose between native runc containers, and gvisor and
+kata-container sandboxes. We might create the following runtime classes:
+
+```yaml
+kind: RuntimeClass
+apiVersion: node.k8s.io/v1alpha1
+metadata:
+    name: native  # equivalent to 'legacy' for now
+spec:
+    runtimeHandler: runc
+---
+kind: RuntimeClass
+apiVersion: node.k8s.io/v1alpha1
+metadata:
+    name: gvisor
+spec:
+    runtimeHandler: gvisor
+----
+kind: RuntimeClass
+apiVersion: node.k8s.io/v1alpha1
+metadata:
+    name: kata-containers
+spec:
+    runtimeHandler: kata-containers
+----
+# provides the default sandbox runtime when users don't care about which they're getting.
```

Comment:
How is the default supplied? Sorry it's been a while since I last read it so this may have already been covered before.

Reply:
In this example, I meant the user just specifies "RuntimeClass = sandboxed", so not actually API defaulting - and this RuntimeClass defines what "sandboxed" means.